docs/docs-content/clusters/pcg/deploy-app-pcg.md

```diff
@@ -0,0 +1,641 @@
+  resource `spectrocloud_privatecloudgateway_ippool` is currently commented out as this tutorial uses dynamic IP
+  placement.
+
+- **terraform.tfvars** - Variable definitions required for the deployment. Open the **terraform.tfvars** file in the
+  editor of your choice and provide the values for the variables according to the table below.
+
+  | Variable Name        | Description                                                                                                 | Required |
+  | -------------------- | ----------------------------------------------------------------------------------------------------------- | -------- |
+  | `metallb_ip`         | Provide a valid IP address or IP range from your VMware environment to be assigned to your load balancer.   | Yes      |
+  | `pcg_name`           | Provide the name of the PCG that will be used to deploy the VMware cluster.                                 | Yes      |
+  | `datacenter_name`    | Provide the name of the vSphere datacenter to target when deploying the VMware cluster.                     | Yes      |
+  | `folder_name`        | Provide the name of the vSphere folder to target when deploying the VMware cluster.                         | Yes      |
+  | `search_domain`      | Provide the name of the network search domain of your vSphere environment. For example, `spectrocloud.dev`. | Yes      |
+  | `vsphere_cluster`    | Provide the vSphere compute cluster to use for the VMware cluster deployment.                               | Yes      |
+  | `datastore_name`     | Provide the vSphere datastore name to use for the VMware cluster deployment.                                | Yes      |
+  | `network_name`       | Provide the vSphere network name to use for the VMware cluster deployment.                                  | Yes      |
+  | `resource_pool_name` | Provide the vSphere resource pool to target when deploying the VMware cluster.                              | Yes      |
+  | `ssh_key`            | Provide the path to your public SSH key. If not provided, a new key pair will be created.                   | No       |
+  | `ssh_key_private`    | Provide the path to your private SSH key. If not provided, a new key pair will be created.                  | No       |
+
+When you are ready making the required changes, initialize Terraform with the command presented below.
+
+```bash
+terraform init
+```
+
+The `init` command will download the plugins and providers defined in the **provider.tf** file.
+
+Next, issue the `terraform plan` command to preview the resources that Terraform will create.
+
+```bash
+terraform plan
+```
+
+The output displays the resources that Terraform will create in an actual implementation, including the cluster profile,
+VMware vSphere cluster, and a new SSH key pair if the keys were not provided.
+
+```text hideClipboard
+# Output condensed for readability
```

Comment:
I've not seen these lines in any of our tutorials.

Reply:
It's not something we have formalized or standardized on but it's out there.